epub33/core/index.html

```diff
@@ -8711,9 +8711,49 @@ <h2>Security and Privacy</h2>
 		<section id="app-overview-unsupported" class="appendix">
 			<h2>Unsupported Features</h2>
 
-			<p>This specification contains certain features the Working Group no longer recommends for use or that are
-				only retained for legacy support reasons. This section defines the meanings of the designations attached
-				to these features and their support expectations.</p>
+			<p>This specification contains certain features that are not yet fully supported in Reading Systems, that
+				the Working Group no longer recommends for use, or that are only retained for interoperability with EPUB
+				2 Reading Systems. This section defines the meanings of the designations attached to these features and
+				their support expectations.</p>
+
+			<section id="risky">
+				<h3>Risky Features</h3>
+
+				<p>A <strong>risky</strong> feature is a feature introduced prior to EPUB 3.3 but that the Working Group
+					has not been able to establish is implemented in two or more Reading Systems as required by <a
+						href="https://www.w3.org/Consortium/Process/">W3C Process</a>.</p>
+
+				<p>These features are considered important to retain despite this limitation because they are known to
+					be implemented by EPUB Creators (i.e., their deprecation would invalidate existing content) and/or
+					they are integral to the content model on which EPUB is built.</p>
+
+				<p>If this specification designates a feature as risky, the following hold true:</p>
+
+				<ul>
+					<li>
+						<p><a>EPUB Creators</a> MAY use the features as described in their <a>EPUB Publications</a>.</p>
```

Comment:
I would remove "as described" (here and in the next item).
I first read "features as-described-in-their-publications" instead of "features-as-described in their publication" and was confused.

Reply:
Ya, I see your point. Maybe it would be better to drop "in their EPUB Publications" as where else are they going to use them? The "as described" is there because you still have to use them as specced, which is a more important point.